dbuild/__init__.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import os
+import shutil
+import glob
+from dbuild import exceptions
+
+from jinja2 import Environment, FileSystemLoader
+from docker import Client
+from tempfile import mkdtemp
+
+def docker_client(url='unix://var/run/docker.sock'):
+    """ return docker client """
+    return Client(url)
+
+def build_image(docker_client, path, tag):
+    """ Build docker image"""
+    return ''.join(['\n'.join(line.values()) for line in docker_client.build(
```

Comment:
Sorry, this seems more confusing than what you had there before.

Why not just:

    return '\n'.join((line.values() for line in ...))

?